### PR TITLE
Add Additional Attributes to Procedure Resource: reasonCode

### PR DESCRIPTION
**Pull request by greenemcg**

### CODE REVIEW

File: app/assets/javascripts/helpers/fhir_value_sets.js.coffee [hunk 1]
```diff
@@ -636,7 +636,8061 @@
       }
     }
 
-
+  # Description   https://hl7.org/fhir/R4/valueset-procedure-reason.html
+  @PROCEDURE_REASON_VS =
```
Comment (ghost) [inline app/assets/javascripts/helpers/fhir_value_sets.js.coffee]: If possible I would suggest moving the value set into a separate file. fhir_value_sets is too big to maintain. Examples can be seen in **helpers/value_sets** files

Comment (greenemcg) [reply app/assets/javascripts/helpers/fhir_value_sets.js.coffee]: I will try also from a UI perspective having 2000 items in a list that is not sorted is a joke

Comment (greenemcg) [reply app/assets/javascripts/helpers/fhir_value_sets.js.coffee:640]: Can u tell me what file or do i have to look through all 22 of them

Comment (ghost) [reply app/assets/javascripts/helpers/fhir_value_sets.js.coffee]: Might make sense to trim the list in that case. Users can enter a custom code anyway.

Comment (greenemcg) [reply app/assets/javascripts/helpers/fhir_value_sets.js.coffee]: that would have to be a separate story by sone one knowing which codes are more important than others.

**Review by adongare — APPROVED** [review]

**Pull request closed without merging**